doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -94,12 +94,92 @@ The ftReceiptCase indicates the receipt type and defines how it should be proces
 For Germany (DE) the country code is `0x4445`. Thus, the value for an unknown ftReceiptCase in Germany is `0x4445000000000000`.
 
 | **Value** | **Description** | **Service- Version** |
-|----------------------|
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------------------|
```

Comment:
set everywhere the version where this is supported from to 1.3.

Reply:
done